sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/VirtualMachinesOperationsExtensions.cs

```diff
@@ -518,13 +528,18 @@ public static async Task<IPage<VirtualMachine>> ListAsync(this IVirtualMachinesO
             /// <param name='operations'>
             /// The operations group for this extension method.
             /// </param>
+            /// <param name='filter'>
+            /// The system query option to filter VMs returned in the response. Allowed
+            /// value is 'virtualMachineScaleSet/id' eq
+            /// /subscriptions/{subId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachineScaleSets/{vmssName}'
+            /// </param>
             /// <param name='statusOnly'>
             /// statusOnly=true enables fetching run time status of all Virtual Machines in
             /// the subscription.
             /// </param>
-            public static IPage<VirtualMachine> ListAll(this IVirtualMachinesOperations operations, string statusOnly = default(string))
+            public static IPage<VirtualMachine> ListAll(this IVirtualMachinesOperations operations, string filter = default(string), string statusOnly = default(string))
```

Comment:
customization

Reply:
@markcowl hello, mark. Is there a customization I can do? 
I know talked about this case last time.  'string' parameter added to a place of another 'string' parameter, and there not much we can do. 

But this is a VirtualMachine List operation and I feel that this may impact more customers than usual

Reply:
resolved by changing the orders on the swagger file. thank you